### PR TITLE
Translate date classes

### PR DESCRIPTION
**Pull request by Fan2Shrek**
Here is the translation of missing date classes:

- `dateError`
- `dateException`
- `dateInvalidOperationException`
- `dateInvalidTimeZoneException`
- `dateMalformedIntervalStringException`
- `dateMalformedPeriodStringException`
- `dateMalformedStringException`
- `dateObjectError`
- `dateRangeError`

### CODE REVIEW

**Comment by devnexen** [comment]
Looking good to me.

File: reference/datetime/dateerror.xml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  <section xml:id="dateerror.intro">
+   &reftitle.intro;
+   <para>
+    Lancée quand la base de données des fuseaux horaires n'est pas trouvée, ou contient des données invalides.
+   </para>
+   <para>
+    Cette erreur ne devrait jamais se produire, et ne dépend pas du code. Il y a deux
+    exceptions enfants (<exceptionname>DateObjectError</exceptionname> et
+    <exceptionname>DateRangeError</exceptionname>) qui sont lancées en fonction
+    des erreurs du programmeur ou des problèmes de plage.
```
Comment (Girgias) [inline reference/datetime/dateerror.xml:20]:
```suggestion
    des erreurs du développeur ou des problèmes de plage.
```
Plutôt non?

File: reference/datetime/dateobjecterror.xml [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+<!-- {{{ DateObjectError intro -->
+  <section xml:id="dateobjecterror.intro">
+   &reftitle.intro;
+   <para>
+    Lancée lorsqu'une des classes Date/Heure n'a pas été correctement
+    initialisée.
+   </para>
+   <para>
+    Parce que les classes Date/Heure ne sont pas finales, ces classes peuvent hériter.
```
Comment (Girgias) [inline reference/datetime/dateobjecterror.xml:18]:
```suggestion
    Parce que les classes Date/Heure ne sont pas finales, ces classes peuvent être hérité.
```
Non?

Comment (Fan2Shrek) [reply reference/datetime/dateobjecterror.xml]: Effectivement c'est plus compréhensible

**Pull request merged**